app/src/main/java/com/ataulm/whatsnext/di/AppComponent.kt

```diff
@@ -18,7 +18,7 @@ import javax.inject.Singleton
         ]
 )
 @Singleton
-interface AppComponent {
+internal interface AppComponent {
```

Comment:
Had to change all these to internal when I konverted WhatsNextService and marked that as internal.

this project only has one module anyway 🙈